{
  DroolsException expectedException=new DroolsException();
  controlCallback.expectAndThrow((mockCallback).doInWorkingMemory(mockWorkingMemory),expectedException);
  mocks.replay();
  try {
    template.execute(mockCallback);
    fail("expected DroolsRuntimeException");
  }
 catch (  DroolsRuntimeException e) {
  }
  mocks.verify();
}
